x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -220,7 +221,7 @@ setup:
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
   - gt: {esql.functions.categorize: $functions_categorize}
-  - length: {esql.functions: 131} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 132} # check the "sister" test above for a likely update to the same esql.functions length check
```

Comment:
this is needed for `last_over_time`, which bumped the sister test but not this one.